dask_jobqueue/tests/test_job.py

```diff
@@ -113,7 +113,7 @@ def test_header_lines_skip():
 
 
 @pytest.mark.asyncio
-async def test_nprocs_scale():
+async def test_nworkers_scale():
```

Comment:
I also changed the name of this test. That might make the CI unhappy.

Reply:
CI problems are unfortunately a bit old. I'm hopping to have a look at it in the following weeks.